packages/@aws-cdk/aws-rds/lib/database-secret.ts

```diff
@@ -28,6 +28,7 @@ export class DatabaseSecret extends secretsmanager.Secret {
   constructor(scope: Construct, id: string, props: DatabaseSecretProps) {
     super(scope, id, {
       encryptionKey: props.encryptionKey,
+      description: `Auto-generated RDS secret for user: ${props.username}`,
```

Comment:
Change this to something like: 'Generated by the CDK for stack: ${stackname}'

Reply:
Will do.